cmd/minishift/cmd/config/config.go

```diff
@@ -78,6 +78,8 @@ var (
 	Metrics           = createFlag("metrics", SetBool, nil, nil, true)
 	Logging           = createFlag("logging", SetBool, nil, nil, true)
 	ServiceCatalog    = createFlag("service-catalog", SetBool, nil, nil, true)
+	// ability to specify additional cluster up flags
+	OcClusterExtraFlags = createFlag("oc-cluster-extra-flags", SetString, nil, nil, true)
```

Comment:
I think the flag name should be `--clusterup-extra-flags`. @praveenkumar @hferentschik ^